lang/de_DE.json

```diff
@@ -0,0 +1,416 @@
+    "You must <1>sign in</1> before you can join this conversation.": "Du musst dich <1>anmelden</1> bevor du diesem Gespräch beitreten kannt.",
+    "You've reached the end of the list.": "Du hast das Ende dieser Liste erreicht.",
+    "You've requested account deletion.": "Du hast deine Accountlöschung beantragt.",
+    "Your Connect API key has been reset.": "Dein Verbindungs-API-Schlüssel wurde zurückgesetzt.",
+    "Your Connect API key is used by emulators to keep you logged in. Resetting the key will log you out of all emulators.": "Dein Connect-API-Schlüssel wird von den Emulatoren verwendet, um dich eingeloggt zu halten. Wenn du den Schlüssel zurücksetzt, wirst du bei allen Emulatoren abgemeldet.",
+    "Your account will be permanently deleted on {{date}}": "Dein Account wird am {{date}} dauerhaft gelöscht",
+    "Your account's personal data will be cleared from the database permanently.": "Die persönlichen Daten deines Accounts werden dauerhaft aus der Datenbank gelöscht.",
+    "Your account's username will NOT be available after the deletion.": "Der Benutzername deines Accounts wird nach der Löschung NICHT mehr verfügbar sein.",
+    "Your personal data is processed by RetroAchievements.org only in accordance with the provisions of applicable data privacy laws. If links on our pages route you to other pages, please inquire there about how your data is handled in such cases.": "Deine persönlichen Daten werden von RetroAchievements.org nur in Übereinstimmung mit den Bestimmungen des geltenden Datenschutzrechts verarbeitet. Wenn Sie über Links auf unseren Seiten auf andere Seiten weitergeleitet werden, erkundige dich bitte dort, wie deine Daten in solchen Fällen behandelt werden.",
+    "Your web API key has been reset.": "Dein Web-API-Schlüssel wurde zurückgesetzt",
+    "by {{authorDisplayName}}": "von {{authorDisplayName}}",
+    "confirm your new email address here...": "Bestätige deine neue E-Mail-Adresse hier...",
+    "confirm your new password here...": "Bestätige dein neues Passwort hier...",
+    "current page number": "Derzeitige Seitennummer",
+    "enter a motto here...": "Gebe hier ein Motto ein...",
+    "enter a new password here...": "Gebe ein neues Passwort hier ein...",
+    "enter your current password here...": "Gebe dein aktuelles Passwort hier ein...",
+    "enter your new email address here...": "Gebe deine neue E-Mail-Adresse hier ein...",
+    "in <1>{{forumTopicTitle}}</1>": "in <1>{{forumTopicTitle}}</1>",
+    "in a few seconds": "in ein paar Sekunden",
+    "in less than a minute": "in weniger als einer Minute",
+    "just now": "gerade eben",
+    "less than a minute ago": "vor weniger als eine Mintute",
```

Comment:
```suggestion
    "less than a minute ago": "vor weniger als eine Minute",
```

Reply:
Must be "vor weniger als eine**r** Minute

Reply:
thanks for noticing this error, correct version should be "vor weniger als einer Minute"